src/IntersonCxxImagingScan2DClass.cxx

```diff
@@ -1,409 +1,424 @@
-    Wrapped->DisposeScan();
-    }
-
-  void AbortScan()
-    {
-    Wrapped->AbortScan();
-    }
-
-  void SetNewBmodeImageCallback( NewBmodeImageCallbackType callback, void * clientData = 0 )
-    {
-    this->BmodeHandler->SetNewBmodeImageCallback( callback, clientData );
-    }
-
-  void SetNewScanConvertedBmodeImageCallback( NewScanConvertedBmodeImageCallbackType callback, void * clientData = 0 )
-    {
-    this->BmodeHandler->SetNewScanConvertedBmodeImageCallback( callback, clientData );
-    }
-
-  void SetNewRFImageCallback( NewRFImageCallbackType callback, void * clientData = 0 )
-    {
-    this->RFHandler->SetNewRFImageCallback( callback, clientData );
-    }
-
-
-private:
-  gcroot< Interson::Imaging::Scan2DClass ^ >                  Wrapped;
-
-  gcroot< BmodeArrayType ^ >                                  BmodeBuffer;
-  gcroot< RFArrayType ^ >                                     RFBuffer;
-  gcroot< NewBmodeImageHandler ^ >                            BmodeHandler;
-  gcroot< NewRFImageHandler ^ >                               RFHandler;
-  gcroot< Interson::Imaging::Scan2DClass::NewImageHandler ^ > BmodeHandlerDelegate;
-  gcroot< Interson::Imaging::Scan2DClass::NewImageHandler ^ > RFHandlerDelegate;
-
-};
-
-
-#pragma unmanaged
-
-Scan2DClass
-::Scan2DClass():
-  Impl( new Scan2DClassImpl() )
-{
-}
-
-
-Scan2DClass
-::~Scan2DClass()
-{
-  delete Impl;
-}
-
-
-bool
-Scan2DClass
-::GetScanOn() const
-{
-  return Impl->GetScanOn();
-}
-
-
-bool
-Scan2DClass
-::GetRFData() const
-{
-  return Impl->GetRFData();
-}
-
-
-void
-Scan2DClass
-::SetRFData( bool transferOn )
-{
-  Impl->SetRFData( transferOn );
-}
-
-
-bool
-Scan2DClass
-::GetFrameAvg() const
-{
-  return Impl->GetFrameAvg();
-}
-
-void
-Scan2DClass
-::SetFrameAvg( bool doAveraging )
-{
-  Impl->SetFrameAvg( doAveraging );
-}
-
-
-void
-Scan2DClass
-::StartReadScan()
-{
-  Impl->StartReadScan();
-}
-
-
-void
-Scan2DClass
-::StartRFReadScan()
-{
-  Impl->StartRFReadScan();
-}
-
-
-void
-Scan2DClass
-::StopReadScan()
-{
-  Impl->StopReadScan();
-}
-
-
-void
-Scan2DClass
-::DisposeScan()
-{
-  Impl->DisposeScan();
-}
-
-
-void
-Scan2DClass
-::AbortScan()
-{
-  Impl->AbortScan();
-}
-
-
-void
-Scan2DClass
-::SetNewBmodeImageCallback( NewBmodeImageCallbackType callback,
-                            void * clientData )
-{
-  Impl->SetNewBmodeImageCallback( callback, clientData );
-}
-
-
-void
-Scan2DClass
-::SetNewScanConvertedBmodeImageCallback( NewScanConvertedBmodeImageCallbackType callback,
-                            void * clientData )
-{
-  Impl->SetNewScanConvertedBmodeImageCallback( callback, clientData );
-}
-
-
-void
-Scan2DClass
-::SetNewRFImageCallback( NewRFImageCallbackType callback,
-                            void * clientData )
-{
-  Impl->SetNewRFImageCallback( callback, clientData );
-}
-
-
-double
-Scan2DClass
-::GetTrueDepth() const
-{
-  return Impl->GetTrueDepth();
-}
-
-} // end namespace Imaging
-
-} // end namespace IntersonCxx
+#pragma unmanaged

```

Comment:
All the lines in this file are marked as changed.  Were the line endings changed from \n?